CONTRIBUTING.md

```diff
@@ -47,7 +47,9 @@ npm linter
 or you can simply run CircleCI locally
```

Comment:
Two lines above this this one it says `npm linter` which is wrong. Can you change that to `npm run lint` please?